examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -21,11 +21,12 @@
      "name": "stdout",
```

Comment:
<p><code>mu1</code>  and <code>mu2</code>  should not have <code>dims="step"</code>,  this is having them both be of shape <code>(2,)</code>.  <code>mu1</code>  should be a scalar and <code>mu2</code>  should have shape <code>(10,)</code>  (you can create and <code>obs</code>  dim to use in <code>mu2</code>),  only <code>accept</code>  has this <code>step</code>  dimension (which is ignored but could be fixed and if so the plot will update the labels automatically).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>